lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -42,11 +52,43 @@ def alto_doc
           output_docs['alto']
         end
 
+        # write the result output files to a given directory
+        def move_result_files(destination_dir)
+          output_dir = Pathname.new(destination_dir)
+          raise "Directory #{output_dir} doesn't exist, please make it first." unless output_dir.directory?
+
+          paths = output_docs.values + split_ocr_paths
+          paths.each do |path|
+            dest_path = File.join(destination_dir, File.basename(path))
+            @logger.info("moving #{path} to #{dest_path}")
+            FileUtils.mv(path, dest_path)
+          end
+        end
+
         private
 
```

Comment:
This finds any XML files that appear to be OCR files split out from the big one.